src/EditorFeatures/CSharpTest/Completion/CompletionProviders/TypeImportCompletionProviderTests.cs

```diff
@@ -33,15 +29,13 @@ internal override CompletionProvider CreateCompletionProvider()
         }
 
         private bool? ShowImportCompletionItemsOptionValue { get; set; } = true;
-
-        // -1 would disable timebox, whereas 0 means always timeout.
-        private int TimeoutInMilliseconds { get; set; } = -1;
+        public object IsExpandedCompletion { get; private set; } = true;
```

Comment:
object?  not bool?

Reply:
Hmm (this is generated by the refactoring)